lib/get-groups.js

```diff
@@ -1,3 +1,5 @@
-module.exports = function getGroups (str) {
-  return str.split(/[:\s+]/)
-}
+"use strict";
+
+module.exports = function getGroups(str) {
+  return str.split(/[:\s+]/);
+};
```

Comment:
Again this is compiled output - it doesn't matter that it's no longer in standard format.